### ISSUE TITLE
MacOS X 10.11 no longer includes OpenSSL

### ISSUE THREAD

**Issue by avsm** [janestreet/async_ssl#7]
So async_ssl fails...

```
#=== ERROR while installing async_ssl.113.00.00 ===============================#
# opam-version 1.2.2
# os           darwin
# command      make
# path         /Users/avsm/.opam/system/build/async_ssl.113.00.00
# compiler     system (4.02.3)
# exit-code    2
# env-file     /Users/avsm/.opam/system/build/async_ssl.113.00.00/async_ssl-14718-d3d859.env
# stdout-file  /Users/avsm/.opam/system/build/async_ssl.113.00.00/async_ssl-14718-d3d859.out
# stderr-file  /Users/avsm/.opam/system/build/async_ssl.113.00.00/async_ssl-14718-d3d859.err
### stdout ###
# [...]
# /Users/avsm/.opam/system/bin/ocamlfind ocamlc -c -g -annot -bin-annot -I bindings -thread -package threads -package ctypes.stubs -package ctypes.foreign.threaded -package ctypes -I bindings -I stubgen -o stubgen/ffi_stubgen.cmo stubgen/ffi_stubgen.ml
# /Users/avsm/.opam/system/bin/ocamlfind ocamlc -linkpkg -g -linkpkg -thread -package threads -package ctypes.stubs -package ctypes.foreign.threaded -package ctypes bindings/async_ssl_bindings.cma stubgen/ffi_stubgen.cmo -o stubgen/ffi_stubgen.byte
# stubgen/ffi_stubgen.byte -c > src/ffi_generated_stubs.c
# /Users/avsm/.opam/system/bin/ocamlfind ocamlc -g -I src -package ctypes -ccopt -I/Users/avsm/.opam/system/lib/ctypes/.. -c src/ffi_generated_stubs.c
# + /Users/avsm/.opam/system/bin/ocamlfind ocamlc -g -I src -package ctypes -ccopt -I/Users/avsm/.opam/system/lib/ctypes/.. -c src/ffi_generated_stubs.c
# src/ffi_generated_stubs.c:2:10: fatal error: 'openssl/ssl.h' file not found
# #include <openssl/ssl.h>
```

I guess we need a Brew version of it.

**Comment by agarwal** [comment]
Has anyone figured out the solution to this. I already have:

```
$ brew install openssl
Warning: openssl-1.0.2d_1 already installed
```

but I still get the same error.

**Comment by rgrinberg** [comment]
@agarwal Not a serious solution but I only use OSX for development so it's good enough for me:

https://twitter.com/rgrinb/status/674438366046261252

**Comment by agarwal** [comment]
Thanks. At least I can continue coding.

**Comment by avsm** [comment]
@diml can this be addressed on OSX in the latest release? It breaks on a lot of local dev setups.

**Comment by diml** [comment]
I'm looking at this

**Comment by diml** [comment]
Fixed in 113.24.00
